docs/ConvertDataOperation.md

```diff
@@ -0,0 +1,110 @@
+# Convert Data operation
+
+FHIR server implements $convert-data, a custom operation to enable data conversion from legacy formats to FHIR format. Currently it supports HL7v2 to FHIR conversion. It uses the Liquid templating engine and default conversion templates from the [FHIR Converter](https://github.com/microsoft/FHIR-Converter/tree/dotliquid) OSS project. 
+
+## How to use $convert-data
+
+Follow these steps to setup and use $convert-data API. Rest of this document details these steps. 
+
+1. Ensure that $convert-data is enabled on the FHIR server.
+1. Call $convert-data endpoint.
+1. Evaluate default templates.
+1. Customize templates if needed.
+1. Publish custom templates.
+1. Make templates accessible to the FHIR Server.
+1. Verify.
+
+### 1. Enable $convert-data
+You can enable $convert-data while deploying FHIR Server. In order to enable or disable the feature at a later stage, set the `FhirServer:Operations:ConvertData:Enabled` setting in the FHIR server to _true_.
```

Comment:
Nit: could you replace the ':'s with '__'. If fhir server is deployed as a web app either syntax will work, but with a docker image deployment only the '__' syntax can be parsed.

Reply:
Sure. That's a good point. Thanks